src/components/ReportActionItem/ReportPreview.tsx

```diff
@@ -119,6 +121,12 @@ function ReportPreview({
         [transactions, iouReportID, action],
     );
 
+    const [isHoldMenuVisible, setIsHoldMenuVisible] = useState(false);
+    const [requestType, setRequestType] = useState<'pay' | 'approve'>();
```

Comment:
Should we create CONST for "pay" and "approve" or use the existing one?

Reply:
I think we don't need to create CONST for this. These values are only the way to help us know what function we will call when click on the button.

Reply:
Let's use REPORT_ACTION_TYPE.PAY and REPORT_ACTION_TYPE.APPROVE

Reply:
I don't think we should use `REPORT_ACTION_TYPE.PAY` and `REPORT_ACTION_TYPE.APPROVE` because the `requestType` here isn't a report action type.

Reply:
yeah, so let's create new CONST to follow our `code of conduct`. We can do as `setPaymentType`

Reply:
@DylanDylann I updated.